util/MsSqlMigratorUtility/packages.lock.json

```diff
@@ -4,7 +4,7 @@
     "net6.0": {
       "Microsoft.Extensions.Logging": {
         "type": "Direct",
-        "requested": "[6.0.0, )",
```

Comment:
It's weird that these are changing. Did you manually do something or what sofware did you use to do nuget restore?